app/grandchallenge/pages/views.py

```diff
@@ -165,8 +166,3 @@ def get_context_data(self, **kwargs):
         )
 
         return context
-
-    def test_func(self):
-        return self.request.user.is_staff or self.request.user.has_perm(
```

Comment:
Missed this before but I think the same about only using `challenges.view_challengerequest`